src/test/scala/viper/gobra/BenchmarkTests.scala

```diff
@@ -47,14 +47,17 @@ trait BenchmarkTests extends StatisticalTestSuite {
 }
 
 trait GobraFrontendForTesting extends Frontend {
+  val z3PropertyName = "GOBRATESTS_Z3_EXE"
+  val z3Exe: Option[String] = Option(System.getProperty(z3PropertyName))
```

Comment:
What is this?

Reply:
This reads the `GOBRATESTS_Z3_EXE` property passed with `-DGOBRATESTS_Z3_EXE`